sass/components/_page-navigation.scss

```
@@ -0,0 +1,25 @@
+.page-navigation {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    margin-bottom: 80px;
+
+    button:nth-child(2) {
```

```suggestion
    button.active {
```
như này chứ, sao lại set cố định bằng nth-child thế
Nhưng những style khác thì cũng hạn chế k nên sử dụng thằng nth-child nhé.
Vì nếu giả sử có bổ sung hay thêm bớt gì thì thứ tự nó sẽ bị đảo lộn hết, và lúc này nth-child k còn đúng nữa